AccessibilityInsightsForAndroidService/app/build.gradle

```
@@ -33,7 +33,62 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
```

Should this line and the above line be updated to use JDK 11?

```suggestion
        targetCompatibility JavaVersion.VERSION_11
```

---

I left those in place intentionally since they're still what the new JDK11-based Android Studio includes in its example projects' build.gradle files. My understanding is that targetCompatibility is talking about "what JVM bytecode should the java compiler emit", and so I'd expect that it would be controlled by which phones we're targetting, rather than which JDK Android Studio comes bundled with